packages/components/psammead-media-player/README.md

```diff
@@ -53,14 +64,15 @@ The `height` and `width` props are required to be provided in advance by AMP so
 ```js
 import { CanonicalMediaPlayer } from '@bbc/psammead-media-player';
 
-const Container = ({ src, title, portrait, showPlaceholder, placeholderSrc }) => (
+const Container = ({ src, title, portrait, showPlaceholder, placeholderSrc, placeholderSrcset, mediaInfo }) => (
```

Comment:
With long lines like this, would it make sense to place one property per line (similar to our Prettier configuration), to improve readability?